apps/api/src/hyp3_api/validation.py

```diff
@@ -77,21 +77,24 @@ def check_dem_coverage(granule_metadata):
 
 
 def check_same_burst_ids(granule_metadata):
-    ref_burst_id, sec_burst_id = [granule['name'].split('_')[1] for granule in granule_metadata]
-    if ref_burst_id != sec_burst_id:
+    burst_ids = [granule['name'].split('_')[1] for granule in granule_metadata]
+    not_matching = [burst_id for burst_id in set(burst_ids) if burst_ids.count(burst_id) != 2]
```

Comment:
My concern with this approach is that we're not actually checking if the two lists of granules were passed in the correct order, we're just checking if each burst ID appears twice in the flattened list of granules. We *could* split the list in half to get the reference and secondary lists, but I'm not sure if we can rely on the order of the granules being preserved after they get passed through `get_cmr_metadata`. I feel like the correct approach would be to refactor our validation approach to be more generalized.